.vsts/python-setup.yml

```diff
@@ -0,0 +1,23 @@
+steps:
+  - task: UsePythonVersion@0
+    condition: eq( variables['Agent.OS'], 'Windows_NT' )
+    inputs:
+      versionSpec: '3.8.x'
+    displayName: Install Python 3.8 for Batch Explorer
+
+  - bash: ./scripts/azpipelines/setup-python.sh
+    displayName: Set up Python environment
+
+  - script: |
+      set -e
+      . "$(Agent.WorkFolder)/.venv/batchexplorer/bin/activate"
+      pip install -r python/requirements.txt
+    condition: ne( variables['Agent.OS'], 'Windows_NT' )
+    displayName: Install Python dependencies
+
+  - powershell: |
+      . .vsts/win/exec.ps1
+      $ErrorActionPreference = "Stop"
+      exec { pip install -r python/requirements.txt }
```

Comment:
The Windows build never used virtualenv before. I'm not even sure what it accomplishes in Linux/Mac given that we're installing Python as part of the build.

Reply:
True, I'm not sure it really buys us anything when using pyenv. I think we should be using it on Windows, though, rather than relying on whatever system Python libraries happen to be installed. Given that I still think it makes sense to use one for uniformity across the builds. If we remove pyenv later (which seems totally reasonable) it would also protect us there.

Reply:
Virtualenv on Windows seems to work just fine. PR updated!